TIPs/TIP-13.md

```diff
@@ -43,6 +43,29 @@ Proposed format:
 
 
 ## Test Cases
+
+Happy path
+ - Sign up
+ - Start season
+ - Start royale in season
+ - Take as position
+ - Claim rewards - 1 player win
+ - Claim rewards - multiple players win
+ - Claim rewards - all players lose
+
+Edge cases
+ - Sign up for Thales Royale with no allowance
+ - Sign up after sign up period has expired
+ - Sign up with already signed up address
+ - Try to start Season before sign up period expired
+ - Try to start royale before seson is started
+ - Try to take a position before royale started.
+ - Try to take a position after round is finished
```

Comment:
try to take the same position twice
try to take a position that is not 1 or 2
try to take position after royale is finished 
try to take a position with players who is not signed up or death in a previous round 
try to take a position when choosing period ends